package.json

```diff
@@ -7,13 +7,13 @@
     "type": "git",
     "url": "https://github.com/oliverschwendener/ueli"
   },
-  "homepage": "https://ueli.oliverschwendener.ch",
+  "homepage": "https://ueli.app",
   "author": {
     "name": "Oliver Schwendener",
     "email": "oliver.schwendener@hotmail.com",
     "url": "https://github.com/oliverschwendener"
   },
-  "description": "A keystroke launcher for Windows and macOS",
+  "description": "This is a keystroke launcher for Windows and macOS.",
   "os": [
     "win32",
     "darwin"
```

Comment:
Your branch is based on `master`, please change that to `dev` too. This is why you get these changes here.

Reply:
I reverted these changes, I'm not sure if this is the right way to do because i don't know how to change the pull request to get commits from the dev branch on my fork.